app/celery/celery.py

```diff
@@ -12,13 +12,19 @@ def worker_process_shutdown(sender, signal, pid, exitcode, **kwargs):
 
 
 def make_task(app):
+    from app import statsd_client
+
     class NotifyTask(Task):
         abstract = True
         start = None
 
         def on_success(self, retval, task_id, args, kwargs):
-            elapsed_time = time.time() - self.start
-            app.logger.info("{task_name} took {time}".format(task_name=self.name, time="{0:.4f}".format(elapsed_time)))
+            task_name = self.name
+            now = time.time()
+            statsd_client.timing_with_dates(f"celery-task.{task_name}.total-time", now, self.start)
```

Comment:
I think this makes sense, but yes lets test in staging based on what you said yes